{{ cookiecutter.project_slug }}/{{ cookiecutter.python_slug }}/__init__.py

```diff
@@ -12,7 +12,7 @@
 __author__ = "{{ cookiecutter.author_name }}"
 __email__ = "{{ cookiecutter.author_email }}"
 
-from .reader_metadata import Reader
+from .reader import Reader
```

Comment:
Question: by importing the reader at the top level, does that mean that in the higher level abstracted away bioio case, when you say `img = BioImage(...)` or similar and the plugin and the reader are determined for you. That it loads ALL readers into memory?